biocrnpyler/dna_construct.py

```diff
@@ -128,7 +127,18 @@ def reverse(self):
         self.name = self.make_name()
         #self.update_base_species()
         return self
-
+    def get_reversed(self):
+        """returns a reversed version of this construct without changing this construct"""
+        outcon = copy.deepcopy(self)
+        outcon.reverse()
+        return outcon
+    def get_circularly_permuted(self,new_first_position):
```

Comment:
This is used when checking if other constructs exist that are the same as a construct that is the product of recombination. If there are, then modify all the reactions so that you are linking the right constructs together. Basically circular construct equality.

Reply:
Is there a construct equality function now?